x-pack/agent/pkg/agent/application/enroll_cmd.go

```diff
@@ -165,11 +171,17 @@ func (c *EnrollCmd) Execute() error {
 	return nil
 }
 
-func metadata() map[string]interface{} {
+func metadata() (map[string]interface{}, error) {
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, err
+	}
+
 	return map[string]interface{}{
 		"platform": runtime.GOOS,
 		"version":  release.Version(),
-	}
+		"host":     hostname,
```

Comment:
Would be good to follow ECS also for this kind of information.

Reply:
^^ @nchaulet

Reply:
Please also take https://www.elastic.co/guide/en/ecs/current/ecs-os.html into account.

Reply:
Do you mean sending the data already as an ECS formatted document if so  I am OK with this proposal.